docs/faq.rst

```diff
@@ -24,18 +35,46 @@ Unfortunately, fMRIPrep's working directory structure is not stable across versi
 so writing code to grab the relevant files from the working directory is a bit of a moving target.
 Nevertheless, we have some code (thanks to Julio Peraza) that works for version 20.2.1.
 
-.. warning::
-    We will try to keep the following gist up-to-date, but there is no guarantee that it will work for a given version.
-    Use it with caution!
-
-    If you do find that the gist isn't working for an fMRIPrep version >= 20.2.1,
-    please comment on `Issue #717 <https://github.com/ME-ICA/tedana/issues/717>`_ (even if it's closed)
-    and we will take a look at the problem.
-
```

Comment:
Given that fMRIPrep added an option to output individual echoes, we don't really need to keep the gist up to date.